components/brave_wallet_ui/common/hardware/ledgerjs/ledger-trusted-transport.ts

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { LedgerMessagingTransport } from './ledger-messaging-transport'
+import {
+  LedgerCommand,
+  AuthorizationSuccessCommand,
+  AuthorizationSuccessResponsePayload
+} from './ledger-messages'
+
+// LedgerTrustedMessagingTransport is the messaging transport object
+// for chrome://wallet and chrome://wallet-panel. It makes calls to the Ledger
+// libraries isolated in chrome-untrusted://ledger-bridge via postMessage,
+// and defines a handler to be run when the untrusted instance is granted access
+// to the user's the Ledger device
```

Comment:
Nice comment here documenting what's going on, here's a nitpick suggestion to give a better understanding of the why aspect as well

```suggestion
// to the user's the Ledger device. This is being done to isolate the ledger bridge
// library from the wallet so that in the event it's compromised it will reduce the 
// impact to our wallet.
```

Reply:
Sounds good to me, though I'll probably put the comment in LedgerUntrustedMessagingTransport instead since that's the code that runs inside the untrusted frame

Reply:
https://github.com/brave/brave-core/pull/14096/commits/de369da43f4d6a22d42b5d5bac93cbf492402b07

Reply:
👍🏻 resolving